src/open_company/app.clj

```diff
@@ -46,6 +47,12 @@
     hot-reload-routes))
 
 (defn start [port]
+  (timbre/set-config! c/log-config)
+  ;; See https://stuartsierra.com/2015/05/27/clojure-uncaught-exceptions
+  (Thread/setDefaultUncaughtExceptionHandler
+   (reify Thread$UncaughtExceptionHandler
+     (uncaughtException [_ thread ex]
+       (timbre/error ex "Uncaught exception on" (.getName thread)))))
```

Comment:
Not really in the scope of this PR but probably just fine it leave it in anyways.

**tl;dr:** Exceptions in threads may not always be handled by default. This snippet makes sure that in every thread there is a handler for uncaught exceptions which just logs them.